spec/ndx-holographic-stimulation.extensions.yaml

```diff
@@ -22,24 +79,32 @@ groups:
     doc: link to the holographic stimulus pattern
     quantity: '?'
 - neurodata_type_def: HolographicSeries
-  neurodata_type_inc: OptogeneticSeries
```

Comment:
Opened this at https://github.com/NeurodataWithoutBorders/nwb-schema/issues/556 to get some help why we can't extend `OptogeneticSeries`

Reply:
Thank you so much!